K2Bridge.Tests.UnitTests/Visitors/HistogramAggregationVisitorTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace UnitTests.K2Bridge.Visitors
```

Comment:
After reviewing the visitor itself.
a) as @tamirkamara  suggested can we filter before ordering? 
b) I am missing tests here, e.g. the 'hard bound' part

Reply:
a) fixed
b) I added the hard bounds and min_doc_count tests